tutorials/vue/bind-src.md

```diff
@@ -0,0 +1,79 @@
+You can dynamically control the image source using the `v-bind:src`, or `:src` for short, binding.
+This allows you to insert JavaScript in the attribute field.
+
+## methods
+
+```javascript
+<script>
+  const { createApp } = Vue
+
+  createApp({
+    methods: {
+        getPhoto() {
+            return '../../assets/logo.png'
+        }
+    },
+    template: `
+    <img style="width:50%; height:50%;" :src="getPhoto()" />
+    `
+  }).mount('#app')
+</script>
+```
+
+
+<script src="https://unpkg.com/vue@3"></script>
+
+<div id="app"></div>
+
+<script>
+  const { createApp } = Vue
+
+  createApp({
+    methods: {
+        getPhoto() {
+            return '../../assets/logo.png'
+        }
+    },
+    template: `
+    <img style="width:50%; height:50%;" :src="getPhoto()" />
+    `
+  }).mount('#app')
+</script>
+
+
+## computed
```

Comment:
Add a quick sentence here about using computed properties for `:src` to explain what the code sample does